Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,129 @@
 incorrect data","24429","Failure","ERROR","Low","Could not establish connection with Active Directory","Could not establish connection with Active Directory","24850","Success","DEBUG","Informational","Successfully connected to external ODBC database","ISE successfully established a new connection to external ODBC database","24851","Failure","DEBUG","Low","Connection to external ODBC database failed","ISE failed to establish a new connection to external ODBC database","34120","Failure","ERROR","Low","Profiler failed to get the connection to NAC Manager","Profiler sends a notification event to NAC Manager, but the notification fails because could not connect to NAC Manager","34147","Failure","WARN","Low","JGroups TLS Handshake Failed","JGroups TLS Handshake Failed","34148","Success","INFO","Informational","JGroups TLS Handshake Succeeded","JGroups TLS Handshake Succeeded","34149","Failure","WARN","Low","HTTPS TLS Handshake Failed","HTTPS TLS Handshake Failed","34150","Success","INFO","Informational","HTTPS TLS Handshake Succeeded","HTTPS TLS Handshake Succeeded","34159","Success","INFO","Informational","LDAPS connection established successfully","LDAPS connection established successfully","34160","Success","INFO","Informational","LDAPS connection terminated successfully","LDAPS connection terminated successfully","34161","Failure","WARN","Low","LDAPS connection establishment failed with SSL error","LDAPS connection establishment failed with SSL error","34162","Failure","WARN","Low","LDAPS connection terminated with SSL error","LDAPS connection terminated with SSL error","34163","Failure","WARN","Low","LDAPS connection establishment failed with non-SSL error","LDAPS connection establishment failed with non-SSL error","34164","Failure","WARN","Low","LDAPS connection terminated with non-SSL error","LDAPS connection terminated with non-SSL error","90062","Failure","ERROR","Low","Cannot connect to Domain Controller","Cannot connect to Domain Controller","90063","Success","INFO","Informational","Successfully establish connection to Domain Controller","Successfully establish connection to Domain Controller","90066","Failure","ERROR","Low","Lost connection with Domain Controller","Lost connection with Domain Controller","90078","Success","INFO","Informational","Closed connection to Domain Controller","Closed connection to Domain Controller","91082","Failure","WARN","Low","RADIUS DTLS: Connection to OCSP server failed","RADIUS DTLS: Connection attempt to OCSP server failed.","11317","Failure","WARN","Low","TrustSec SSH connection failed","ISE failed to establish SSH connection to a network device. Verify network device SSH credentials in the Network Device page are similar to the credentials configured on the network device. Check network device enabled ssh connections from ISE (ip address)","5405","Failure","NOTICE","Low","RADIUS Request dropped","RADIUS request dropped","5406","Failure","NOTICE","Low","TACACS+ Request dropped","TACACS+ request dropped"];
```

Comment:
Based on the event, there should be DvcAction in Network Session as well.
For some of the initial records of LDAP, the events do not look like NetworkSession. They are more of Audit events.

Reply:
Added DvcAction, lookup list updated